src/main/java/workout/workoutmanager/WorkoutManagerParser.java

```diff
@@ -91,13 +106,18 @@ public static int parseIndex(String args) throws NotANumberException {
         return index;
     }
 
+    /**
+     * Parses user input into a list of predicates which will be used to limit period of record being listed.
+     * @param args user input
```

Comment:
can consider adding a blank line between description and @param to follow [coding standards](https://se-education.org/guides/conventions/java/basic.html#comments)? And add punctuation behind each parameter description also. It's in the "note in particular" session.

Reply:
Icic. Will check on that